### PR TITLE
release: Prepare for 0.3.2

### PR DESCRIPTION
**Pull request by real-or-random**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.3.2] - 2023-05-12
+We strongly recommend updating to 0.3.2 if you use or plan to use GCC >=13 to compile libsecp256k1. When in doubt, check the Clang version using `clang -v`.
```
Comment (jonasnick) [inline CHANGELOG.md:9]: check the version with gcc -v

**Review by jonasnick — COMMENTED** [review]
ACK 074e83cb8ea230f785fcb2a84e95a95f5fc236ad

**Comment by real-or-random** [comment]
Applied the following diff to mention ABI compatibility with 0.3.0 and update the date to tomorrow.

```
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 649132b0..6c5dbb84 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.3.2] - 2023-05-12
+## [0.3.2] - 2023-05-13
 We strongly recommend updating to 0.3.2 if you use or plan to use GCC >=13 to compile libsecp256k1. When in doubt, check the GCC version using `gcc -v`.
 
 #### Security
@@ -22,7 +22,7 @@ We strongly recommend updating to 0.3.2 if you use or plan to use GCC >=13 to co
  - Renamed asm build option `arm` to `arm32`. Use `--with-asm=arm32` instead of `--with-asm=arm` (GNU Autotools), and `-DSECP256K1_ASM=arm32` instead of `-DSECP256K1_ASM=arm` (CMake).
 
 #### ABI Compatibility
-The ABI is compatible with version 0.3.1.
+The ABI is compatible with versions 0.3.0 and 0.3.1.
 
 ## [0.3.1] - 2023-04-10
 We strongly recommend updating to 0.3.1 if you use or plan to use Clang >=14 to compile libsecp256k1, e.g., Xcode >=14 on macOS has Clang >=14. When in doubt, check the Clang version using `clang -v`.
```

**Comment by hebasto** [comment]
I've made [check installations](https://github.com/bitcoin-core/secp256k1/pull/1310/):

- Autotools:
```
$ tree $dir
/tmp/tmp.vsAYM7Fe9E
├── include
│   ├── secp256k1_ecdh.h
│   ├── secp256k1_extrakeys.h
│   ├── secp256k1.h
│   ├── secp256k1_preallocated.h
│   └── secp256k1_schnorrsig.h
└── lib
    ├── libsecp256k1.a
    ├── libsecp256k1.la
    ├── libsecp256k1.so -> libsecp256k1.so.2.0.2
    ├── libsecp256k1.so.2 -> libsecp256k1.so.2.0.2
    ├── libsecp256k1.so.2.0.2
    └── pkgconfig
        └── libsecp256k1.pc

3 directories, 11 files
```

- CMake:
```
$ tree $dir
/tmp/tmp.C1t6Ydgg1q
├── include
│   ├── secp256k1_ecdh.h
│   ├── secp256k1_extrakeys.h
│   ├── secp256k1.h
│   ├── secp256k1_preallocated.h
│   └── secp256k1_schnorrsig.h
└── lib
    ├── cmake
    │   └── libsecp256k1
    │       ├── libsecp256k1-config.cmake
    │       ├── libsecp256k1-config-version.cmake
    │       ├── libsecp256k1-targets.cmake
    │       └── libsecp256k1-targets-relwithdebinfo.cmake
    ├── libsecp256k1.so -> libsecp256k1.so.2
    ├── libsecp256k1.so.2 -> libsecp256k1.so.2.0.2
    └── libsecp256k1.so.2.0.2

4 directories, 12 files
```

**Comment by hebasto** [comment]
I've [updated](https://github.com/hebasto/secp256k1-CMake-example/tree/88b0f45e2bc30172a9c0c184a587aa1c883593cb) a CMake's example of integration `secp256k1` into a top project as its source subtree, which is the way how it will be used in Bitcoin Core.

<details>
<summary>CMakeLists.txt</summary>

```CMake
cmake_minimum_required(VERSION 3.13)

project(secp256k1-example LANGUAGES C)

set(SECP256K1_DISABLE_SHARED ON)
set(SECP256K1_ENABLE_MODULE_RECOVERY ON)
set(SECP256K1_BUILD_BENCHMARK OFF)
add_subdirectory(src/secp256k1)

add_executable(${PROJECT_NAME} src/ecdsa.c)
target_link_libraries(${PROJECT_NAME}
  secp256k1
  $<$<PLATFORM_ID:Windows>:bcrypt>
)

include(GNUInstallDirs)
install(TARGETS ${PROJECT_NAME}
  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
)
```

</details>

It looks much cleaner than it was [before](https://github.com/hebasto/secp256k1-CMake-example/blob/38bbc83075730ca1f4960de5120a0e2cf04f29a3/CMakeLists.txt).

cc @theuni

**Review by Coins2payroll — APPROVED** [review]

**Review by hebasto — APPROVED** [review]
ACK d490ca2046be118258c6a0617c3461c913d1f208

**Review by jonasnick — COMMENTED** [review]
ACK d490ca2046be118258c6a0617c3461c913d1f208

**Review by scottwad — APPROVED** [review]

**Pull request merged**